packages/terra-heading/src/terra-dev-site/doc/example/colors.css

```diff
@@ -0,0 +1,22 @@
+:root {
+  --terra-heading-red-color: rgb(220, 20, 60);
+  --terra-heading-green-color: rgb(46, 125, 50);
+  --terra-heading-blue-color: #304ffe;
+  --terra-heading-black-color: #333;
+}
+
+.crimson {
+  color: var(--terra-heading-red-color);
+}
+
+.green {
+  color: var(--terra-heading-green-color);
+}
+
+.blue {
+  color: var(--terra-heading-blue-color);
+}
+
+.black {
+  color: var(--terra-heading-black-color);
+}
```

Comment:
Consumers are probably going to copy paste this file when they begin development and swap values / classes. 

Is this how we would expect them to utilize a class approach? Should we encourage them to take a more direct theming approach with fallback syntax? 

We're removing color to avoid issues with themes. I think we should abstract the class/variable names away from the colors.


```scss
.default {
  color: var(--terra-heading-default-color, #fff);
}

.attention {
  color: var(--terra-heading-attention-color,  rgb(220, 20, 60));
}

.neutral {
  color: var(--terra-heading-neutral-color, rgb(46, 125, 50));
}
```

Reply:
Fixed for terra-heading and terra-text here https://github.com/cerner/terra-core/pull/2211/commits/92b70a437ee68ee43c5fdd288a5fd4be5298ab22

Reply:
Fixed terra-progress-bar to not use colors in the class naming scheme for the examples here https://github.com/cerner/terra-core/pull/2211/commits/0902ee68e1682870c54ba3e913cd18c136c7ce3a